qdrant-landing/content/documentation/faq/qdrant-fundamentals.md

```diff
@@ -81,3 +81,35 @@ We only guarantee compatibility if you update between consecutive versions. You
 
 In case your version is older, we only guarantee compatibility between two consecutive minor versions. This also applies to client versions. Ensure your client version is never more than one minor version away from your cluster version.
 While we will assist with break/fix troubleshooting of issues and errors specific to our products, Qdrant is not accountable for reviewing, writing (or rewriting), or debugging custom code.
+
+### Which embedding models does FastEmbed support, and how does it utilize memory?
+
+You can find the list of supported models [here](https://qdrant.github.io/fastembed/examples/Supported_Models/).
+
+Memory usage in fastEmbed depends on several factors:
+1. **Size of the Text**: Everything, including the text and its vectors, is loaded into memory. As vectors are computed, RAM consumption increases accordingly.
+2. **Data Parallelism**: fastEmbed utilizes Python's multiprocessing to split large lists of strings into smaller ones and process them in parallel. While this enhances processing speed, it can also lead to higher RAM consumption.
```

Comment:
where is the source of this info?

Reply:
https://github.com/qdrant/fastembed/issues/222#issuecomment-2104016782

Reply:
I am not sure if this is correct. Data parallelism might require multiple copies of the model per each process. But it have nothing to do with splitting the list of texts